spec/python-spec.coffee

```diff
@@ -296,3 +296,28 @@ describe "Python grammar", ->
       expect(tokens[1][0]).toEqual value: 'SELECT bar', scopes: ['source.python', scope]
       expect(tokens[2][0]).toEqual value: 'FROM foo', scopes: ['source.python', scope]
       expect(tokens[3][0]).toEqual value: delim, scopes: ['source.python', scope, 'punctuation.definition.string.end.python']
+
+
+  it "tokenizes SQL inline highlighting on blocks with a CTE", ->
+    delimsByScope =
+      "string.quoted.double.block.sql.python": '"""'
+      "string.quoted.single.block.sql.python": "'''"
+
+    for scope, delim in delimsByScope
```

Comment:
Yes, this needs to be `of`.  I've fixed the other test in #103 if you want to take a look.